cmd/shp/main.go

```diff
@@ -15,8 +17,8 @@ import (
 const ApplicationName = "shp"
 
 func main() {
-	flags := pflag.NewFlagSet(ApplicationName, pflag.ExitOnError)
-	pflag.CommandLine = flags
+	init_goflags()
+	init_pflags()
```

Comment:
Style nit - go functions typically don't use underscores in names. This is more of a convention thing than anything enforced in the language:
```suggestion
	initGoFlags()
	initPFlags()
```